documentation/src/docs/asciidoc/docinfos/index-docinfo.html

```diff
@@ -0,0 +1 @@
+<link rel="icon" type="image/png" href="{{ site.baseurl }}/img/junit5-logo.png">
```

Comment:
@marcphilipp, sorry, my bad. The command that I used was `gitPublishCommit` which generated the `gitpublish` folder at the same level with the `gh-pages` and there in `/docs/current/api` is the `index.html` for the javadocs and in `/docs/current/api` is the `index.html` for the userguide. The favicons appear for both of them.